apps/emqx_machine/src/emqx_machine_boot.erl

```diff
@@ -148,9 +148,13 @@ basic_reboot_apps() ->
     ?BASIC_REBOOT_APPS ++ (BusinessApps -- excluded_apps()).
 
 excluded_apps() ->
-    OptionalApps = [bcrypt, jq, observer],
-    [system_monitor, observer_cli] ++
-        [App || App <- OptionalApps, not is_app(App)].
+    %% Apps that _should never_ be (re)started automatically:
+    ExcludedApps = [system_monitor, observer_cli, jq],
```

Comment:
@kjellwinblad This will make sure jq app is _never_ started automatically. Is it the right thing to do?